File: javascript-closure-angular/API/Client/RealmDomain.js
goog.provide('API.Client.RealmDomain');

/**
 * Object containing details of the newly added domain. 
 * @record
 */
API.Client.RealmDomain = function() {}

/**
 * The new allowed domain. 
 * @type {!string}
 * @export
 */
API.Client.RealmDomain.prototype.domain;

/**
 * Whether subdomains are allowed for this domain. 
 * @type {!boolean}
 * @export
 */
API.Client.RealmDomain.prototype.allowSubdomains;

File: scala-play-server/app/model/RealmDomain.scala
package model

import play.api.libs.json._

/**
  * Object containing details of the newly added domain. 
  * @param domain The new allowed domain. 
  * @param allowSubdomains Whether subdomains are allowed for this domain. 
  */
@javax.annotation.Generated(value = Array("org.openapitools.codegen.languages.ScalaPlayFrameworkServerCodegen"))
case class RealmDomain(
  domain: Option[String],
  allowSubdomains: Option[Boolean]
)

object RealmDomain {
  implicit lazy val realmDomainJsonFormat: Format[RealmDomain] = Json.format[RealmDomain]
}

